run-local.sh

```diff
@@ -19,7 +19,8 @@ restart_docker () {
   #Prune existing containers
   #Comment in if you wish to perform a fresh install of all containers where all containers are removed and deleted
   #You will be prompted to continue with the deletion in the terminal
-  #docker system prune --all
+
+  docker system prune --all
```

Comment:
this is worth keeping commented or whacking behind and optional flag as it's takes a long time to run